content/projects/crystallize.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Crystallize
+homepage: https://www.crystallize.com/
+twitter: CrystallizeAPI
+repo: https://github.com/CrystallizeAPI
+opensource: "No"
+typeofcms: "API Driven"
+supportedgenerators:
+    - All
+description: Crystallize is a GraphQL PIM service with a twist; manage structured product information and rich marketing content. A fast GraphQL API for building juicy headless customer experiences in any channel. Ideal for JAMStack & React. Sign up to your [free tier Crystallize account](https://crystallize.com/signup) and start building.
+images:
+    - path: /images/crystallize-JAMStack-ecommerce-boilerplate.png
+    - path: /images/crystallize-beautiful-editorial-ui.png
+    - path: /images/crystallize-topic-maps-taxonomies.png
+    - path: /images/crystallize-fast-graphql-api.png
+---
+
+<img class="simple" src="/images/crystallize-beautiful-editorial-ui.png" alt="Crystallize Admin UI" />
+
+## Bring your own frontend
+
+Build your own tailor made frontend or get started with our open source JAMStack boilerplate:
+
+<img class="simple" src="/images/crystallize-JAMStack-ecommerce-boilerplate.png" alt="Furniture Shop on Crystallize" />
+
+```bash
+npx @crystallize/cli <project-name>
+```
+
+More information on how to start in our [learn section](https://crystallize.com/learn)
+
+Check out how details of how to build your own headless storefront or content site using our boilerplate for [JAMStack eCommerce with Next.js and Vercel](https://crystallize.com/learn/open-source/boilerplates/react-nextjs)
+
+## Key Features
+
+Crystallize is a [PIM service](https://crystallize.com/product/product-information-management) with a twist. Manage super structured product information and rich marketing content in one. Build websites, webshops, Apps and more with a fast, easy to use API and a beautiful admin UI.
+
+Ideal for [JAMStack & React](https://crystallize.com/blog/beautiful-and-fast-jamstack-ecommerce-in-3-steps).
+
+### GraphQL API
+
+All created content is accessible via a [GraphQL API](https://crystallize.com/blog/realtime-pim-api-for-super-fast-ecommerce).
+
+<img class="simple" src="/images/crystallize-fast-graphql-api.png" alt="GraphQL API on Crystallize" />
+
+### Scalable cloud
+
+Your content is stored on our servers, they will be safe, secure and delivered super fast. Images are transcoded and converted to fast WebP formats. HD video is transcoded and ready for streaming direct from our Edge network.
+
+### Structured Content
+
+Model content your way using [shapes](https://crystallize.com/learn/concepts/pim/shape) in Crystallize. Build a [custom content model](https://crystallize.com/blog/structured-content-pim-withhttps://crystallize.com/learn/best-practices/information-architecture/content-modelling) instead of dealing with fixed, predefined templates.
```

Comment:
`https://crystallize.com/blog/structured-content-pim-withhttps://crystallize.com/learn/best-practices/information-architecture/content-modelling` link doesn’t work

Reply:
Update the link